Formula/jvm-mon.rb

```diff
@@ -4,7 +4,7 @@ class JvmMon < Formula
   url "https://github.com/ajermakovics/jvm-mon/releases/download/0.3/jvm-mon-0.3.tar.gz"
   sha256 "9b5dd3d280cb52b6e2a9a491451da2ee41c65c770002adadb61b02aa6690c940"
   license "Apache-2.0"
-  revision 2
+  revision 3
```

Comment:
Not sure if `revision` bump or rebuild preferred for replacing `all` bottle. The difference between Linux vs macOS library will probably require separate OS bottles.